spec/controllers/users_controller_spec.rb

```diff
@@ -1440,6 +1442,20 @@ def post_user
         end
       end
 
+      context 'for an activated account with unconfirmed email' do
+        it 'should send an email' do
+          unconfirmed_email_user = Fabricate(:user, active: true)
+          email_token = unconfirmed_email_user.email_tokens.create(email: unconfirmed_email_user.email).token
```

Comment:
Oh yeah, that's correct. I just want to create an email token here. Removed the variable.